subsys/net/ip/net_context.c

```diff
@@ -229,7 +229,8 @@ int net_context_get(sa_family_t family,
 			continue;
 		}
 
-		if (ip_proto == IPPROTO_TCP) {
+		if (!IS_ENABLED(CONFIG_NET_OFFLOAD) &&
```

Comment:
As we could have both offloaded and normal network interfaces in the system, you need to do the checking same way as it is already in net_context.c. So please add `net_if_is_ip_offloaded()` call into this check.

Reply:
added